murenn/dtcwt/nn.py

```diff
@@ -38,38 +43,64 @@ def __init__(self, *, J, Q, T, in_channels, padding_mode="symmetric"):
                 groups=in_channels,
                 padding="same",
             )
-            torch.nn.init.normal_(conv1d_j.weight)
+            torch.nn.init.normal_(conv1d_j.weight, std=1/math.sqrt(T))
```

Comment:
what's the rationale for this `1/sqrt(T)`? i'm guessing it has to do with expected value of the random energy response

are you testing it somewhere?

Reply:
no, I didn't do any tests, this was a mistake.